src/pivot-table/hooks/use-column-width.ts

```diff
@@ -142,6 +146,7 @@ export default function useColumnWidth(
     return {
       leftGridWidth: Math.min(rect.width * LEFT_GRID_MAX_WIDTH_RATIO, sumOfWidths),
       leftGridColumnWidths: columnWidths,
+      leftGridFullWidth: columnWidths.reduce((acc, curr) => acc + curr, 0),
```

Comment:
This should always be the same as `sumOfWidths?`

Reply:
In which case you might as well expose it as `allLeftGridColumnsVisible` here instead

Reply:
Yes, they are same, but for naming i think it should stay like as it is (`leftGridFullWidth`) since we have it's counterpart for right grid as well at here: https://github.com/qlik-oss/sn-pivot-table/blob/472e5ae2b958312641ecad224f002c1a88299fe2/src/pivot-table/hooks/use-column-width.ts#L293